File: insertion_sort/insertionSort.js
function insertionSort (nums) {
    for(let i = 1; i < nums.length; i++){
      for(let j = 0; j < i; j++) {
        console.log(nums + ' I = ' + i + ' J = ' + j )
        // 5 < 10
        if(nums[i] < nums[j]){
         // [5]
          const spliced = nums.splice(i,1);
         // [5,10..]
          nums.splice(j,0,spliced[0])
        }
      }
    }
   return nums;
  }

File: insertion_sort/insertionSort.py
def insertionSort(nums):
    index = 1
    for i in range(index, len(nums)):
        for j in range(0, index):
            if nums[i] < nums[j]:
                insert = nums.pop(i)
                nums.insert(j, insert)
        if index is not len(nums):
            index += 1
    return nums
